air/tests/test_module/features/misc/version_check.rs

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use air::PreparationError;
+use air_test_utils::prelude::*;
+
+#[test]
```

Comment:
i would add a test for our branch versions. it seems this will break them, cuz they're usually 0.0.1-foo-bar

Reply:
Good catch